lib/ruby_lsp/requests/rename.rb

```diff
@@ -72,7 +72,7 @@ def perform
 
         # If the client doesn't support resource operations, such as renaming files, then we can only return the basic
         # text changes
-        unless @global_state.supported_resource_operations.include?("rename")
+        unless @global_state.client_capabilities.supported_resource_operations.include?("rename")
```

Comment:
(nit) Could add a method `ClientCapabilities#supports_rename?` so there's one less level to drill through.

Reply:
Good idea. Added that method.